code/stable/dblpendulum/SRS/HTML/DblPendulum_SRS.html

```diff
@@ -167,22 +167,22 @@ <h2>Table of Symbols</h2>
                 </tr>
                 <tr>
                   <td><em>a<sub>x1</sub></em></td>
-                  <td>The horizontal acceleration of the first object</td>
+                  <td>Horizontal acceleration of the first object</td>
```

Comment:
So this change is by design? I like it, I prefer this without the extra leading `The`.

Reply:
This was the only change that was supposed to be committed, I accidentally branched off of my working branch for another issue.